react-ui/src/pages/submission/SubmissionForm.jsx

```diff
@@ -36,8 +36,14 @@ const API_PATH = '/submission';
 const MAX_FILE_SIZE = 1024 * 1024; // 1MB
 const BACKGROUND_PADDING = 20;
 const BACKGROUND_BOX_SIZE = 75;
+const CHAR_LIMIT_NAME = 80;
+const CHAR_LIMIT_DESCRIPTION = 1200;
 
 const useStyles = makeStyles((theme) => ({
+  alignRight: {
```

Comment:
_Nit_: An issue with naming styles after the style they apply, e.g. bold, alignRight, etc. is that you end up adding things that don't fit to it, e.g. `paddingLeft` here, isn't really needed to align something to the right, we're using it because of where we are applying this class. In the future, this might not be apparent, and we could try to use the `alignRight` class to strictly an element to the right and have unexpected results. CSS classes should instead be named after what they are being applied to, so this would probably be better called something like `characterCounter`.